packages/block-library/src/latest-posts/edit.js

```diff
@@ -262,6 +280,13 @@ class LatestPostsEdit extends Component {
 						categorySuggestions={ categorySuggestions }
 						onCategoryChange={ selectCategories }
 						selectedCategories={ categories }
+						onAuthorChange={ ( value ) =>
+							setAttributes( {
+								users: '' !== value ? value : undefined,
```

Comment:
We should make sure we save the ID as number, it is string by default, so:
`users: '' !== value ? Number( value ) : undefined`

Reply:
I agree. I've updated the code.